client/state/posts/utils.js

```diff
@@ -249,3 +251,12 @@ export function normalizePostForApi( post ) {
 
 	return normalizeApiFlow( post );
 }
+
+/**
+ * Check if the content is empty (ignoring empty tags)
+ * @param  {string}  content Raw post content
+ * @return {Boolean}         Whether it's considered empty
+ */
+export function isEmptyContent( content ) {
```

Comment:
Curious how we might handle these types of utility functions once selectors are all moved to the `state/selectors` directory. I personally don't really like them all that much, but seem an unfortunate necessity in some cases. Here, since `isEmptyContent` is probably only needed by the `editedPostHasContent` selector, I might imagine it could live alongside the selector in its `state/selectors/edited-post-has-content.js` file (exported only for purposes of testing, but not to be used elsewhere).